pubsub-adapter/src/main/kotlin/com/ably/pubsub/Presence.kt

```diff
@@ -0,0 +1,46 @@
+package com.ably.pubsub
+
+import io.ably.lib.types.*
+
+/**
+ * Enables the retrieval of the current and historic presence set for a channel.
```

Comment:
We're missing `get` here if we want this comment to be truthful

Reply:
I'll update comment, unfortunately rest and realtime presence have different API for get